### DISCUSSION TITLE
Need some help

### DISCUSSION THREAD

**Discussion by wvneville** [apprenticeharper/DeDRM_tools#1478] [Q&A]
Won't remove DRM - What's the problem?  Info copied below.  Thanks

Microsoft Windows [Version 10.0.19041.685]
(c) 2020 Microsoft Corporation. All rights reserved.

C:\Users\Wayne>calibre-debug -g
calibre 5.9  embedded-python: True is64bit: False
Windows-10-10.0.19041 Windows ('32bit', 'WindowsPE')
32bit process running on 64bit windows
('Windows', '10', '10.0.19041')
Python 3.8.5
Windows: ('10', '10.0.19041', '', 'Multiprocessor Free')
Interface language: None
Successfully initialized third party plugins: Gather KFX-ZIP (from KFX Input) (1, 36, 0) && DeDRM (7, 0, 3) && Package KFX (from KFX Input) (1, 36, 0) && KFX metadata reader (from KFX Input) (1, 36, 0) && KFX Input (1, 36, 0) && KePub Output (3, 4, 3) && Kobo Utilities (2, 12, 0)
Turning on automatic hidpi scaling
devicePixelRatio: 2.0
logicalDpi: 96.0 x 96.0
physicalDpi: 45.920903954802256 x 45.87290969899665
Using calibre Qt style: True
[0.00] Starting up...
[0.00] Showing splash screen...
[0.36] splash screen shown
[0.36] Initializing db...
[0.39] db initialized
[0.39] Constructing main UI...
DEBUG:    0.0 KoboUtilites::action.py - loading translations
DEBUG:    0.0 KoboUtilites::dialogs.py - loading translations
DEBUG:    0.0 KoboUtilites::action.py - loading translations
DEBUG:    0.6 No Kobo Touch, Glo or Mini appears to be connected
DEBUG:    0.6 rebuild_menus - self.supports_ratings=None, self.supports_tiles=None
DEBUG:    0.6 KoboUtilities:set_toolbar_button_tooltip - start: text='None'
DEBUG:    0.6 device_driver_name - could not load extended driver. Exception= No module named 'calibre_plugins.kobotouch_extended'
DEBUG:    0.6 KoboUtilities:set_toolbar_button_tooltip - setting to text='Utilities to use with Kobo ereaders

Driver: KoboTouch'
DEBUG:    0.6 device_driver_name - could not load extended driver. Exception= No module named 'calibre_plugins.kobotouch_extended'
[1.12] main UI initialized...
[1.12] Hiding splash screen
Starting QuickView
[1.41] splash screen hidden
[1.41] Started up in 1.41 seconds with 453 books
stdout+stderr from file dialog helper: [b'', b'']
piped data from file dialog helper: [b'\x8fm\x14\xc3l\xbb-\xff9\xc5e\xea\xf5\x98:%!\x9c\xfb\xdd\xb2h\xd0\x1b\xbe\xd2\x93.\x91\x88\x19\xf8', b'C:\\Users\\Wayne\\Documents\\My Kindle Content\\B08QND25GL_EBOK Unreported truths\\B08QND25GL_EBOK.azw']
Gather KFX-ZIP (from KFX Input) 1.36.0: Importing C:\Users\Wayne\Documents\My Kindle Content\B08QND25GL_EBOK Unreported truths\B08QND25GL_EBOK.azw
Gather KFX-ZIP (from KFX Input): Cannot locate some files needed for the book. KFX files must remain in their original folder structure for successful import.
Gather KFX-ZIP (from KFX Input): Gathered 1 file(s) as C:\Users\Wayne\AppData\Local\Temp\calibre_ye_eftx2\qb0q7n15.kfx-zip
DeDRM v7.0.3: Trying to decrypt qb0q7n15.kfx-zip
Using Library AlfCrypto DLL/DYLIB/SO
Using Library AlfCrypto DLL/DYLIB/SO
Decrypting KFX-ZIP ebook: qb0q7n15
Got DSN key from database default_key
Found 1 keys to try after 0.1 seconds
DeDRM v7.0.3: Failed to decrypt with error: The .kfx-zip archive contains an encrypted DRMION file without a DRM voucher
DeDRM v7.0.3: Looking for new default Kindle Key after 0.1 seconds
searching for kinfoFiles in C:\Users\Wayne\AppData\Local
Found K4PC 1.25+ kinf2018 file: C:\Users\Wayne\AppData\Local\Amazon\Kindle\storage\.kinf2018
Decrypted key file using IDString '3326130213' and UserName 'Wayne'
DeDRM v7.0.3: Found 1 new key
Decrypting KFX-ZIP ebook: qb0q7n15
Got DSN key from database default_key_1
Found 1 keys to try after 3.5 seconds
DeDRM v7.0.3: Ultimately failed to decrypt after 3.5 seconds. Read the FAQs at Harper's repository: https://github.com/apprenticeharper/DeDRM_tools/blob/master/FAQs.md
Package KFX (from KFX Input) 1.36.0: Packaging C:\Users\Wayne\AppData\Local\Temp\calibre_ye_eftx2\qb0q7n15.kfx-zip
Processing container: B08QND25GL_EBOK.azw
KFX metadata reader (from KFX Input) activated for C:\Users\Wayne\AppData\Local\Temp\calibre_ye_eftx2\wsbirspr_add_books\0\B08QND25GL_EBOK.kfx-zip
Added B08QND25GL EBOK to db in: 0.1
Added 1 books in 4.3 seconds
Traceback (most recent call last):
  File "calibre\gui2\jobs.py", line 170, in update
KeyboardInterrupt
Traceback (most recent call last):
  File "calibre\gui2\jobs.py", line 170, in update
KeyboardInterrupt

**Comment by ElleKayEm** [comment]
The beta DeDRM isn't handling KFX format yet.  See the methods in this post for avoiding KFX and getting books in the older format:  https://www.mobileread.com/forums/showthread.php?t=283371
